parameters.yml

```diff
@@ -51,7 +51,7 @@ demeAmplitudes: [0.1, 0., 0.1]              # amplitude of seasonality
 demeOffsets: [0., 0., 0.5]                  # seasonal offset relative to the year
     
                                             # immune parameters  
-phenotypeSpace: "geometric"                 # options include: geometric, geometric3d, geometric10d
+phenotypeSpace: "sequence"                 # options include: geometric, geometric3d, geometric10d, sequence
```

Comment:
Do we want to keep the original simulation parameter set but provide a separate new one?

Reply:
That's a good idea - thanks!